website/docs/docs/build/measures.md

```diff
@@ -12,7 +12,7 @@ Measures are aggregations performed on columns in your model. They can be used a
 | --- | --- | --- |
 | [`name`](#name) | Provide a name for the measure, which must be unique and can't be repeated across all semantic models in your dbt project. | Required |
 | [`description`](#description) | Describes the calculated measure. | Optional |
-| [`agg`](#aggregation) | dbt supports the following aggregations: `sum`, `max`, `min`, `count_distinct`, and `sum_boolean`. | Required |
+| [`agg`](#aggregation) | dbt supports the following aggregations: `sum`, `min`, `max`, `average`, `sum_boolean`, `count_distinct`, `median`, and `percentile`. | Required |
```

Comment:
```suggestion
| [`agg`](#aggregation) | dbt supports aggregations such as `sum`, `min`, `max`, and others. Refer to [Aggregation](/docs/build/measures#aggregation) for more supported aggregation types. | Required |
```

Reply:
good catch again @b-per and i've simplified it so it directs the user to the dedicated section for more info on support agg types. thank you for opening this up!